cmd/loadtest/main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"math"
```

Comment:
compile the project at least once before submitting. golang forces alphabetical ordering in imports